src/components/layout/MobileDashboard.tsx

```diff
@@ -0,0 +1,164 @@
+import React from "react";
```

Comment:
Is it standard practice to have another file for Mobile (just asking)?

Reply:
Yes, if the designs for mobile dashboard and desktop dashboard are different.